Assets/VRTK/Scripts/VRTK_InteractGrab.cs

```diff
@@ -37,6 +37,8 @@ public class VRTK_InteractGrab : MonoBehaviour
         public float grabPrecognition = 0f;
         [Tooltip("An amount to multiply the velocity of any objects being thrown. This can be useful when scaling up the `[CameraRig]` to simulate being able to throw items further.")]
         public float throwMultiplier = 1f;
+        [Tooltip("Automatically scale the throw velocity up and down to match the scale of the `[CameraRig]`.")]
+        public bool throwScalesWithPlayer = true;
```

Comment:
Do you want to close this PR and just do a new one with the one word change? I'm not fussed which way round it's done :)